tests/selenium/test_tooltip.py

```diff
@@ -138,3 +139,77 @@ def test_ttip003_tooltip_disappears(test):
 
     wait.until(lambda: tooltip.exists(), 2.5)
     wait.until(lambda: not tooltip.exists(), 2.5)
+
+
+def test_ttip004_tooltip_applied(test):
+    props = {
+        **base_props,
+        "columns": [
+            dict(
+                id=str(i),
+                name=["Column {}".format(math.ceil(i / 3)), "Column {}".format(i)],
+            )
+            for i in range(1, 30)
+        ],
+        "merge_duplicate_headers": True,
+        "tooltip_delay": 0,
+        "tooltip_duration": 5000,
+        "tooltip": {
+            "1": "text1",
+            "2": {"use_with": "data", "value": "text2"},
+            "3": {"use_with": "header", "value": "text3"},
+        },
+        "tooltip_data": [{"1": "alt-text1"}],
+        "tooltip_header": [{}, {"1": "alt-header1-row1"}],
+    }
+
+    app = dash.Dash(__name__)
+    app.layout = DataTable(**props)
```

Comment:
Test combination of static tooltips with/without usage and data/header tooltips to make sure usage is applied and overrides override.